src/frontend/README.md

```diff
@@ -1,7 +1,128 @@
-# frontend
+# frontend service
 
-Run the following command to restore dependencies to `vendor/` directory:
+The **frontend** service is responsible for rendering the UI for the store's website.
+It serves as the main entry point for the application routing requests to their
+appropriate backend service.
+The application uses Server Side Rendering (SSR) to generate HTML consumed by
+the browser.
 
-```sh
-dep ensure --vendor-only
+## OpenTelemetry features
+
+| Auto-Instrumented tracing | Custom spans | Custom span attributes | Metrics | Logs |
```

Comment:
yes, using an instrumentation library.

Reply:
This term has been used differently a lot. I was hoping the addition of the glossary would help https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#automatic-instrumentation

But I too continue to misuse it :) I actually thought things like middlewares were still considered "auto instrumentation", but the glossary says otherwise.

Reply:
I'd suggest to use InstrumentationLibrary, as it is the official term, and auto-instrumentation is different.
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#instrumentation-libraries